### DISCUSSION TITLE
Kodi ClearArt, ClearLogo, DiscArt, etc. with PKC

### DISCUSSION THREAD

**Discussion by OGTK423** [croneter/PlexKodiConnect#1914] [Q&A]
Hi there. Been using PKC with Kodi for a while now and am a huge fan.

Is there a way to take advantage of Kodi's additional artwork types such as ClearArt, ClearLogo, DiscArt, etc. when using PKC? 

The Kodi skin I'm using (Aeon Nox Silvo v8.0.1) automatically grabs some of these but I'd like to manually override them with art of my own collection but I'm not sure how to go about doing this. In Kodi on it's own, it appears you would add the art to the same folder as the movie and give it a specific file name.

I'm not sure how you'd go about doing that when using PKC.

Appreciate any assistance anyone might be able to provide!

**Comment by hamakavula** [comment]
Doesn't this work by default if you choose "Download additional images and information online" in the initial PKC setup???
_Never noticed before, I had to make a workaround for myself the other day to DISABLE these clearart and fanarts... xD_

---
Luckely I have this fully working and totally automated with **'Ember Media Manager'** for many years now _(previously in Kodi standalone(s) and now with Plex and PKC combined)_ https://github.com/DanCooper/Ember-MM-Newscraper/releases.

Check this _(using Windows)_:

1. **Labeling files** is **VERY** important to automate to whole thing but no biggy, check here for (Plex/Kodi/EmberMediaManager-)details https://support.plex.tv/articles/naming-and-organizing-your-tv-show-files/
_Example movie "D:\Movies\Terminator Collection\Terminator 2 Judgement Day\Terminator 2 Judgement Day.mkv"_
_Example episode "D:\Series\Tour of Duty\Season 01\Tour of Duty S01E01.mkv"_

2. **Download and install Ember ** takes about 10 minutes to setup _(Just check all settings and all tabs and boxes to finish ful setup)._

3. Get Ember to start scraping _and all folders get filled with images and info (that Kodi standalone also handles (offline))._

4. Get a proper skin (recommending "Estuary MOD v2" for Nexus very very very much for making Kodi more awesomer than awesome _(and full PKC functionality)_ https://repo.kodinerds.net/

Sidenote: **Ignoring (folder(s)):** Plex _(and therefore Kodi+PKC)_ ignores all images and fields wherever edited _(locked fields in Plex)._ And all files/folders containing "*.plexignore" in the name.


Voilá! Full (offline) image library in combination with Kodi and PKC!

**Comment by OGTK423** [comment]
Thanks for the write up.

Is Ember Media Manager _required_ for what you just describe? I just sort of went all in on TinyMediaManager and would rather not jump to another metadatas program if avoidable.

**Reply by hamakavula** [reply]
I'm not familiar with TinyMediaManager... sounds the same but smaller 😄

Anyways, at least as the app downloads all images (posters, fanart, clearart, etc.) then Plex will pass them on through Kodi via PKC.

But I would first try the Skin, it also fetches a lot of (extra) data and images...

**Comment by croneter** [comment]
https://github.com/croneter/PlexKodiConnect/issues/1936?notification_referrer_id=NT_kwDOAEbE9LI1Nzg0ODE5NTU1OjQ2Mzc5NDA

**Reply by gshpychka** [reply]
404
